src/app/shared/components/prompt/prompt-fields.ts

```diff
@@ -150,133 +150,6 @@ export function INVITATION_FIELDS(initialEmail?: string): PromptField[] {
   return fields;
 }
 
-export const CREDIT_CARD_FIELDS: PromptField[] = [
```

Comment:
Ooh, fantastic catch, I didn't notice those when I spent a few minutes on this last Friday!